ehrapy/preprocessing/_imputation.py

```diff
@@ -353,12 +312,12 @@ def miss_forest_impute(
     and using the trained model to predict the missing values.
 
     See https://academic.oup.com/bioinformatics/article/28/1/112/219101.
-    This requires the computation of which columns in X contain numerical only (including NaNs) and which contain non-numerical data.
+
+    If required, the data needs to be properly encoded as this imputation requires numerical data only.
 
     Args:
         adata: The AnnData object to use MissForest Imputation on.
-        var_names: List of columns to impute or a dict with two keys ('numerical' and 'non_numerical') indicating which var
-                   contain mixed data and which numerical data only.
+        var_names: List of columns to impute
```

Comment:
```suggestion
        var_names: Iterable of columns to impute
```

You changed the type to `Iterable` so we can also note this here.